po/ja.po

```diff
@@ -14411,12 +14447,17 @@ msgid ""
 "implement `Future` for your own types. For example, the `JoinHandle` returned\n"
 "from `tokio::spawn` implements `Future` to allow joining to it."
 msgstr ""
+"非同期の関数は`impl Future`を返却します。あなた専用の型に対して`Future`を実装することも（よくあるこ"
+"とではありませんが）可能です。例えば、`tokio::spawn`から返却される`JoinHandle`は`Future`を実装する"
+"ことにより、結果の結合を可能にしています。"
```

Comment:
Here, "joining" is a technical term in async context. So, it may be okay to leave it untranslated.